workers/unity/Packages/packages-lock.json

```diff
@@ -0,0 +1,389 @@
+{
```

Comment:
This file is explicitly ignored by the [`gitignore`](https://github.com/spatialos/gdk-for-unity/blob/develop/workers/unity/.gitignore#L10).

How did you manage to check this in? In any case, this shouldn't be here.

Reply:
I think I didn't pull the develop branch before I made this branch 🤦

Reply:
Ahhhh, that would explain it 😂